specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,959 @@
+      "x-ms-parameter-location": "method"
+    },
+    "CollectionId": {
+      "name": "collectionId",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "description": "Identifier (name) of a specific collection.",
+      "x-ms-parameter-location": "method"
+    },
+    "FeatureId": {
+      "name": "featureId",
+      "in": "path",
+      "type": "string",
+      "description": "Case sensitive identifier of a specific feature.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "FeatureBody": {
+      "in": "body",
+      "name": "feature",
+      "required": true,
+      "description": "A valid `GeoJSON Feature` object type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.2) for details.",
+      "x-ms-parameter-location": "client",
+      "schema": {
+        "$ref": "#/definitions/Feature"
+      }
+    },
+    "FeatureAttributesBody": {
+      "in": "body",
+      "name": "featureAttributes",
+      "required": true,
+      "description": "The geometry and properties of a feature.",
+      "x-ms-parameter-location": "client",
+      "schema": {
+        "$ref": "#/definitions/FeatureAttributes"
+      }
+    }
+  },
+  "paths": {
+    "/features/datasets/{datasetId}": {
+      "get": {
+        "description": "The Get Landing Page API provides links to the API definition, the Conformance statements, and the metadata about the feature data in this dataset.\n\nThe Features API is a part of [Creator](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps), which makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset/create). Datasets consist of multiple feature collections, which are a collection of features of a similar type based on a common schema referred to as ontology. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).",
```

Comment:
I don't believe that page can be customized, at least I haven't seen any of our docs doing that. We had a previous discussion about the redundancy before and decided to keep it, so that anyone that stumbles upon a specific page can have a reference to others. See https://github.com/Azure/azure-rest-api-specs/pull/19539#discussion_r903177508